Kickstarter-iOS/Views/Storyboards/ProjectPamphlet.storyboard

```diff
@@ -19,7 +19,7 @@
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
                             <containerView opaque="NO" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" translatesAutoresizingMaskIntoConstraints="NO" id="tIm-aj-d8Q">
-                                <rect key="frame" x="0.0" y="52" width="400" height="1348"/>
+                                <rect key="frame" x="0.0" y="55" width="400" height="1346"/>
```

Comment:
Had to move the ProjectPamphletContent container down a little bit because it was overlapping the top cell.